packages/riipen-ui/src/components/Form.test.jsx

```diff
@@ -0,0 +1,247 @@
+    });
+
+    it("renders error", () => {
+      const error = "Yay";
+
+      const wrapper = mount(<Form error={error} />);
+
+      expect(wrapper.find("Form").text()).toEqual(error);
+    });
+
+    it("renders error within Typography element if error is a string", () => {
+      const error = "Yay";
+
+      const wrapper = mount(<Form error={error} />);
+
+      expect(wrapper.find("Typography").contains(error)).toEqual(true);
+    });
+
+    it("renders error as given element", () => {
+      const error = <Typography>Please provide a name</Typography>;
+
+      const wrapper = mount(<Form error={error} />);
+
+      expect(wrapper.find("Typography").contains(error)).toEqual(false);
+      expect(wrapper.find("Form").props().error.type).toEqual(Typography);
+    });
+  });
+
+  describe("errors prop", () => {
+    const props = {};
+
+    const tests = [
+      { argument: ["Error", "Sad"] },
+      { argument: { errorOne: "Error", errorTwo: "Sad" } },
+      {
+        argument: [
+          <Typography>Please provide a name</Typography>,
+          <Typography>Error</Typography>
+        ]
+      }
+    ];
+
+    tests.forEach(test => {
+      it("sets class name as errors with valid custom errors type", () => {
+        props.errors = test.argument;
+
+        const wrapper = mount(<Form errors={props.errors} />);
+
+        expect(
+          wrapper
+            .find("Notice")
+            .childAt(0)
+            .childAt(0)
+            .hasClass("errors")
+        ).toEqual(true);
+      });
+    });
+
+    it("sets class name as errorContainer when valid errors are used", () => {
+      const errors = ["Error", "Sad"];
+
+      const wrapper = mount(<Form errors={errors} />);
+
+      expect(
+        wrapper
+          .find("Form")
+          .childAt(0)
+          .hasClass("errorContainer")
+      ).toEqual(true);
+    });
+
+    it("does not set errorContainer class name without valid errors", () => {
+      const wrapper = mount(<Form />);
+
+      expect(
+        wrapper
+          .find("Form")
+          .childAt(0)
+          .hasClass("errorContainer")
+      ).toEqual(false);
+    });
+
+    it("renders errors from given string array", () => {
+      const errors = ["Error", "Sad"];
+
+      const wrapper = mount(<Form errors={errors} />);
+
+      expect(
+        wrapper
+          .find(".errors")
+          .childAt(0)
+          .childAt(0)
+          .childAt(0)
+          .childAt(0)
+          .text()
+      ).toEqual(errors[0]);
+      expect(
+        wrapper
+          .find(".errors")
+          .childAt(1)
+          .childAt(0)
+          .childAt(0)
+          .childAt(0)
+          .text()
+      ).toEqual(errors[1]);
+    });
+
+    it("renders errors within Typography element if errors are strings", () => {
+      const errors = ["Error", "Sad"];
+
+      const wrapper = mount(<Form errors={errors} />);
+
+      expect(wrapper.find("Typography").contains(errors[0])).toEqual(true);
+    });
```

Comment:
I feel like these are testing the same thing? Do we need both tests?

Reply:
I'll take it out, it's meant to test that an error is rendered within a Typography element, but that's already covered in the error prop block.